frontend_tests/node_tests/hotkey.js

```diff
@@ -62,6 +62,7 @@ const overlays = mock_esm("../../static/js/overlays", {
     drafts_open: () => false,
```

Comment:
Same commit in #21672

Reply:
Cool, this is merged so we can drop this on rebase.